bin/import_lingua_libre.py

```diff
@@ -76,49 +108,19 @@ def _maybe_convert_sets(target_dir, extracted_data):
     for record in glob(glob_dir, recursive=True):
         record_file = record.replace(ogg_root_dir + os.path.sep, '')
         if record_filter(record_file):
-            samples.append((record_file, os.path.splitext(os.path.basename(record_file))[0]))
+            samples.append((os.path.join(ogg_root_dir, record_file), os.path.splitext(os.path.basename(record_file))[0]))
 
-    # Keep track of how many samples are good vs. problematic
-    counter = {'all': 0, 'failed': 0, 'invalid_label': 0, 'too_short': 0, 'too_long': 0, 'total_time': 0}
-    lock = RLock()
+    counter = get_counter()
     num_samples = len(samples)
     rows = []
 
-    def one_sample(sample):
-        """ Take a audio file, and optionally convert it to 16kHz WAV """
-        ogg_filename = path.join(ogg_root_dir, sample[0])
-        # Storing wav files next to the ogg ones - just with a different suffix
-        wav_filename = path.splitext(ogg_filename)[0] + ".wav"
-        _maybe_convert_wav(ogg_filename, wav_filename)
-        file_size = -1
-        frames = 0
-        if path.exists(wav_filename):
-            file_size = path.getsize(wav_filename)
-            frames = int(subprocess.check_output(['soxi', '-s', wav_filename], stderr=subprocess.STDOUT))
-        label = label_filter(sample[1])
-        with lock:
-            if file_size == -1:
-                # Excluding samples that failed upon conversion
-                counter['failed'] += 1
-            elif label is None:
-                # Excluding samples that failed on label validation
-                counter['invalid_label'] += 1
-            elif int(frames/SAMPLE_RATE*1000/10/2) < len(str(label)):
-                # Excluding samples that are too short to fit the transcript
-                counter['too_short'] += 1
-            elif frames/SAMPLE_RATE > MAX_SECS:
-                # Excluding very long samples to keep a reasonable batch-size
-                counter['too_long'] += 1
-            else:
-                # This one is good - keep it for the target CSV
-                rows.append((wav_filename, file_size, label))
-            counter['all'] += 1
-            counter['total_time'] += frames
-
     print("Importing ogg files...")
     pool = Pool(cpu_count())
     bar = progressbar.ProgressBar(max_value=num_samples, widgets=SIMPLE_BAR)
-    for i, _ in enumerate(pool.imap_unordered(one_sample, samples), start=1):
+    for i, processed in enumerate(pool.imap_unordered(one_sample, samples), start=1):
```

Comment:
Instead of accessing processed as a tuple, destructure it here:

`for i, (processed_counter, processed_rows) in enumerate...`

Reply:
I'll switch to that, I was supposed to do it but forgot